Header header
int robot_id
string task_type
string problem 
